docs/architecture/adr-007.md

```diff
@@ -31,7 +32,7 @@ In this ADR, we define generic event argument that can be extended to accept new
 event __CronosSendToIbc(address sender, string recipient, uint256 amount);
 
 // Proposal
-event __CronosSendToIbc(address sender, string recipient, uint256 amount, string channelId, bytes extraData);
+event __CronosSendToIbc(address indexed sender, uint256 indexed channelId, string recipient, uint256 amount, bytes extraData);
```

Comment:
I end up making recipient "non" indexed

reason is that indexed event for variable type store alwayys the hash 
https://docs.soliditylang.org/en/v0.8.16/abi-spec.html#encoding-of-indexed-event-parameters

it means that we cannot retrivee the real value from indexed parameters